api/api-job-instance-v1/src/main/java/com/thoughtworks/go/apiv1/jobinstance/JobInstanceControllerV1.java

```diff
@@ -36,12 +35,14 @@
 
 import java.io.IOException;
 
+import static com.thoughtworks.go.server.service.PipelineHistoryService.BAD_CURSOR_MSG;
```

Comment:
should we find a better home for this constant?

Reply:
What about in `ApiController` ?

Reply:
Not moving it in `ApiController` as the same is also accessed in Controller classes. Creating a Constant class to store the relevant constants